src/components/Tab/TabParts.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
```

React import는 필요하지 않으니 제거하면 좋을 것 같습니당

Reply:
제 컴퓨터에서만 그런지 모르겠는데, 이 부분은 import React를 지우면 아래와 같은 경고메시지가 나와서 import React를 냅뒀습니다.
[여기](https://velog.io/@iberis/React%EB%8A%94-UMD-%EC%A0%84%EC%97%AD%EC%9D%84-%EC%B0%B8%EC%A1%B0%ED%95%98%EC%A7%80%EB%A7%8C-%ED%98%84%EC%9E%AC-%ED%8C%8C%EC%9D%BC%EC%9D%80-%EB%AA%A8%EB%93%88%EC%9E%85%EB%8B%88%EB%8B%A4)를 참고했는데 React를 import 하는것으로 해결되어 그대로 냅뒀어요!

![image](https://github.com/depromeet/10mm-client-web/assets/67476544/81ab1f8c-a2eb-479b-8a40-13c55b3fcac8)